src/binding/c/datatype_api.txt

```diff
@@ -111,7 +111,7 @@ MPI_Pack:
     message.
 */
 { -- error_check -- inbuf, outbuf
-    if (incount > 0) {
+    if (incount > 1) {
```

Comment:
The logic becomes mysterious. How about add a comment, e.g. "MPI_BOTTOM can't be used when count > 1"?

Reply:
I added an according comment.